src/intTest/resources/fhir/pathology.fhir.json

```diff
@@ -116,6 +116,37 @@
           "collectedDateTime": "2010-02-23T15:41:00+00:00"
         }
       }
+    },
+    {
+      "fullUrl": "urn:uuid:0d124e8d-6447-44e8-9d7d-9f5595b1a0c9",
+      "resource": {
+        "resourceType": "Observation",
+        "id": "0d124e8d-6447-44e8-9d7d-9f5595b1a0c9",
+        "status": "final",
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
```

Comment:
Where did this system value come from?

Reply:
https://www.hl7.org/fhir/observation-example-f001-glucose.json.html

Reply:
Don't forget to look at the correct spec when you're looking at FHIR docs (https://www.hl7.org/fhir/STU3/observation-example-f001-glucose.json.html). But value is the same anyway 😄